xAPI.md

```diff
@@ -2789,8 +2789,15 @@ See [Appendix G: Cross Domain Request Example](#AppendixG) for an example.
 
 ## 2.0 Data Storage and Retrieval
 
+This section contains implementation details and requirements surrounding the specifics of data storage within an 
+LRS and retrieval from that LRS.  As mentioned in the previous section, this communication is done through HTTP 
+Requests to specific Resources, all of which have Endpoints. 
+
 ### 2.1 Documents
 
+Documents are a common storage and retrieval mechanism in many of the xAPI Resources.  This section details their 
+use.
+
```

Comment:
Yes, wow that was badly worded.  I should say "medium" or as you say "data type".  I'll reword and work both in.